content/php-client/getting-started.md

```diff
@@ -5,6 +5,21 @@
 * PHP >= 5.6
 * Composer
 
+## Compatibility matrix
+
+Below you will find which PHP client version to use depending on your PIM version.
+
+| PIM version | API PHP client version |
```

Comment:
I will reverse the order in the github for consistency  (PIM version first)

Reply:
I changed the order because I think it's better to have the PIM versions first. You have a PIM version and you ask yourself which version of the PHP client you should use, no?

Reply:
Agree, I said it as a note for myself ;)

Reply:
Ah ok. :)